.editorconfig

```diff
@@ -0,0 +1,20 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+#indentation
+[*.{html,scss,js,yml,json}]
+indent_style = space
+indent_size = 2
```

Comment:
Looks like most of these files have 4 spaces. Can you update to reflect the pre-existing conventions?

Reply:
@jpetto  whats about the indent_size of  .toml and jenkinsfile if it will be 4 then will add in [*] ?

Reply:
`.toml` and `Jenkinsfile` should both have an `indent_size` of 4, so yes, they can both fall under the `[*]` selector.

Currently the `Jenkinsfile` has a mix of 2 and 4 space indentation. It would be great if you could standardize the `Jenkinsfile` in this PR to have 4 spaces.

Reply:
I'm not really picky on this one. Whatever you think is best works. 👍